sangwon/app.js

```diff
@@ -0,0 +1,46 @@
+require("dotenv").config();
+
+const http = require("http");
+
+const express = require("express");
+const cors = require("cors");
+const morgan = require("morgan");
+
+const { DataSource } = require("typeorm");
+
+const mysqlDataSource = new DataSource({
+  type: process.env.TYPEORM_CONNECTION,
+  host: process.env.TYPEORM_HOST,
+  port: process.env.TYPEORM_PORT,
+  username: process.env.TYPEORM_USERNAME,
+  password: process.env.TYPEORM_PASSWORD,
+  database: process.env.TYPEORM_DATABASE,
+});
+
+myDataSource.initialize()
+  .then(() => {
+  console.log("Data Source has been initialized!");
+  })
+  .catch((err)=>{
+    console.log("Error during Data Source initialization",err)
+  })
```

Comment:
해당 코드에서 database와 pooling 즉, 연결하는 부분이 정상적으로 동작하나요? 서버를 열었을 때 "Data Source has been initialized!" 라는 console.log가 찍히면 제대로 연결이 됐다는 것인데, 코드를 보았을 때는 console.log("Error during Data Source initialization",err) 이 부분이 출력 될 것 같네요.

항상 code를 작성한 뒤에는 server가 정상적으로 돌아가는지 확인 후 push 하는 습관을 가져주세요!

```suggestion
const appDataSource = new DataSource({
  type: process.env.TYPEORM_CONNECTION,
  host: process.env.TYPEORM_HOST,
  port: process.env.TYPEORM_PORT,
  username: process.env.TYPEORM_USERNAME,
  password: process.env.TYPEORM_PASSWORD,
  database: process.env.TYPEORM_DATABASE,
});

appDataSource.initialize()
  .then(() => {
  console.log("Data Source has been initialized!");
  })
  .catch((err)=>{
    console.log("Error during Data Source initialization",err)
  })
```